docs/rules/number-to-fixed-digits.md

```diff
@@ -0,0 +1,37 @@
+# Enforce using the digits argument with `Number#toFixed()`
+
+It's better to make it clear what the value of the `digits` argument is when calling [Number#toFixed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed).
```

Comment:
We should mention the default here too, like https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1294